Subject: [PATCH] libpam: add missing DESTDIR prefix

The DESTDIR prefix is missing, this will cause build failures for
mkdir /var/run/sepermit on the host.

| mkdir -p /var/run/sepermit
| mkdir: cannot create directory `/var/run/sepermit': Permission denied

Upstream-Status: Pending

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 modules/pam_sepermit/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules/pam_sepermit/Makefile.am b/modules/pam_sepermit/Makefile.am
index cfc5594..bc82275 100644
--- a/modules/pam_sepermit/Makefile.am
+++ b/modules/pam_sepermit/Makefile.am
@@ -35,7 +35,7 @@ if HAVE_LIBSELINUX
   securelib_LTLIBRARIES = pam_sepermit.la
 
 install-data-local:
-	mkdir -p $(sepermitlockdir)
+	mkdir -p $(DESTDIR)$(sepermitlockdir)
 endif
 if ENABLE_REGENERATE_MAN
 noinst_DATA = README pam_sepermit.8 sepermit.conf.5
-- 
1.7.5.4

